scripts/eda.py

```diff
@@ -27,6 +27,28 @@ def main(processed_dir, results_dir):
     # Load the dataset from the specified directory
     data_adult = pd.read_csv(processed_dir)
 
+    # 要描述的分类变量
```

Comment:
Non-english comments. And I think we can remove this code as its not adding anything to the script steps (only printing data frame). Can you elaborate? This feels like a copy-pasted code from ChatGPT

Reply:
So sorry. I remembered I already deleted the changes to this file, but I don’t know why it’s still updated in the repo.